src/SFA.DAS.ReferenceData.Types/DTO/GetOrganisationResult.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ReferenceData.Types.DTO
```

Comment:
Does this need to be removed from file directory as its not used in the solution anymore?